content/en/docs/Configuration/p8s-jaeger-grafana/tracing/tempo.md

```diff
@@ -336,3 +337,23 @@ spec:
 ```
 
 To configure a secret to be used as a password, see this [FAQ entry]({{< relref "../../../FAQ/installation#how-can-i-use-a-secret-to-pass-external-service-credentials-to-the-kiali-server" >}})
+
+### Tempo cache
+
+Kiali 2.2 includes a simple tracing cache for Tempo that storages the last N elements. By default, it is enabled and it keeps the last 200 traces. It can be modified in the Kiali CR with: 
```

Comment:
```suggestion
Kiali 2.2 includes a simple tracing cache for Tempo that stores the last N traces. By default, it is enabled and it keeps the last 200 traces. It can be modified in the Kiali CR with: 
```